### ISSUE TITLE
Non-scrollable Grid doesn't render the required accessibility attributes 

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#3636]
Reported in private support thread: **1559632**

**Describe the bug**
In scrollable Grid, the generated HTML includes _role="presentation"_ attribute on some Grid elements. But when setting _scrollable="none"_ those attributes are missing which causes accessibility issues.

**To Reproduce**
https://stackblitz.com/edit/angular-brft5z

**Expected behavior**
The _role_ attribute should be rendered regardless of the scrolling state.

**Comment by slavenai** [comment]
This is expected because the rendering of a scrollable and a non-scrollable Grid differ, thus the described difference in accessibility attributes.
